### DISCUSSION TITLE
도커 설치시 permission denied(장철원 강사님 교안)

### DISCUSSION THREAD

**Discussion by GeonSoon1** [WHATDO-TEAM/whatdo#22] [트러블슈팅]
# <트러블슈팅>
```
# 도커 명령어를 유저 모드에서도 입력할수 있도록 설정
# 이렇게 하면 해당 사용자는 sudo 없이도 Docker 명령어를 실행할 수 있게 됩니다.
sudo usermod -aG docker $USER
```

장철원 강사님 교안을 보고 따라했고 위의 코드를 넣고 docker version치니까 permission denied오류가 떴다.

![image (13)](https://github.com/user-attachments/assets/df56882a-eb37-459e-a5ca-e449c279496d)

# <해결>
```
#/var/run/docker.sock 파일의 권한을 666으로 변경하니 정상 실행되었다.
sudo chmod 666 /var/run/docker.sock
```
![image (14)](https://github.com/user-attachments/assets/abc1ca8b-b15d-4983-b871-90f7c2600e84)

**Comment by Kdreamtomaster** [comment]
https://github.com/WHATDO-TEAM/whatdo/discussions/17

이부분 한번 참고 부탁드리겠습니다 맨 마지막줄 입니다.
